app/src/main/res/values/preferences.xml

```diff
@@ -62,6 +62,14 @@
     <string name="preference_title_fast_swiping_enabled">Enable fast swiping</string>
     <string name="preference_summary_fast_swiping_enabled">Allows swiping over multiple rooms.</string>
 
+    <!-- Schedule update notification -->
+    <string name="preference_title_change_statistics_ui_enabled">Enable change statistics dialog</string>
+    <string name="preference_summary_change_statistics_ui_enabled">View change statistics when schedule is updated</string>
+
+    <!-- Schedule lockscreen view -->
+    <string name="preference_title_schedule_lockscreen_view_enabled">Enable schedule lockscreen view</string>
+    <string name="preference_summary_schedule_lockscreen_view_enabled">View the schedule when device is locked</string>
```

Comment:
`View the schedule when device is locked` -> `Allows to interact with the app even when the device is locked.`